src/main/java/com/d_d/praktyki/dice/Dice.java

```diff
@@ -0,0 +1,25 @@
+package com.d_d.praktyki.dice;
+
+import java.util.Random;
+public class Dice {
+    int sides;
+    Random rand = new Random();
+    public Dice(){
+        sides = 6;
+    }
+    public Dice(int sides){
+        this.sides = sides;
+    }
+    public int roll(){
+        return rand.nextInt(sides)+1;
+    }
+    public void roll(int rolls) {
+         rand.ints(1, sides+1)
+                .limit(rolls)
+                .forEach(element ->
```

Comment:
co to znaczy ?

Reply:
Jakie są potencjalne skutki wywołania metody roll(int rolls) z parametrem rolls równym -5 przed i po wprowadzeniu zaproponowanej przez Ciebie zmiany w kodzie? Jakie zmiany w zachowaniu metody można zaobserwować?